compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -39,12 +39,14 @@ class Compiler {
    *   - after erasure, asSeenFrom is the identity, so every reference has a
    *     plain SymDenotation, as opposed to a UniqueRefDenotation.
    */
-  def phases: List[List[Phase]] =
+  def phases(implicit ctx: Context): List[List[Phase]] =
     List(
       List(new FrontEnd),           // Compiler frontend: scanner, parser, namer, typer
       List(new sbt.ExtractDependencies), // Sends information on classes' dependencies to sbt via callbacks
       List(new PostTyper),          // Additional checks and cleanups after type checking
       List(new sbt.ExtractAPI),     // Sends a representation of the API of classes to sbt via callbacks
+      List(macros.newTransform:_*), // Transform macros definitions, may be Nil when scala.meta.eden not in -toolpath
```

Comment:
In scalac, we've been saying `macro definitions`, `macro implementations`, etc, using a singular noun "macro" if it precedes another noun.